env.template

```diff
@@ -12,3 +12,10 @@ HEALTH_CHECK_MAX_RETRY_INTERVAL_SECONDS=10
 HEALTH_CHECK_MAX_RETRIES=4
 CAPACITY_LIMIT='{"type":"percentage", "value":80}'
 
+IPFS_ENDPOINT="http://127.0.0.1:5001"
+IPFS_BASIC_AUTH_USER=""
+IPFS_BASIC_AUTH_SECRET=""
+IPFS_GATEWAY_URL="http://127.0.0.1:8080/ipfs/[CID]"
+
+# should be enabled for e2e tests
+# ENABLE_DEV_CONTROLLER=true
```

Comment:
question: how can we enforce a specific .env file for e2e tests?

Reply:
we can add one for e2e and in the script we can copy it to .env such that node can source it